libcxx/test/libcxx/selftest/gen.cpp/empty.gen.cpp

```diff
@@ -8,4 +8,4 @@
 
 // Make sure we can generate no tests at all
 
-// RUN: true
```

Comment:
Didn't you get a similar problem with `libcxx/test/libcxx/selftest/sh.cpp/run-success.sh.cpp`?

I think we could use `:` instead -- it's a builtin Lit command as well and it is more obvious that it's meant to be used as `true`.

Reply:
The same problem happens with `libcxx/test/libcxx/selftest/sh.cpp/run-success.sh.cpp`, but the failure does not show up during test discovery since it's not a `gen` test, so I didn't spot that. Thanks for pointing that out!